src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -50,6 +59,17 @@ public void parseCommand_add() throws Exception {
         assertEquals(new AddCommand(student), command);
     }
 
+    @Test
+    public void parseCommand_addGroup() throws Exception {
+        Group group = new GroupBuilder().withName(VALID_GROUP_TUTORIAL).build();
+        AllocDescriptor descriptor = new AllocDescriptorBuilder()
+                .withGroup(VALID_GROUP_TUTORIAL)
+                .withName(VALID_NAME_AMY).build();
+        AddGroupCommand command = (AddGroupCommand) parser.parseCommand(AddGroupCommand.COMMAND_WORD + " "
```

Comment:
I don't think you should be casting here. It should be the test's job to ensure that it's the right type.